specs-go/config.go

```diff
@@ -469,3 +479,77 @@ type Syscall struct {
 	Action Action `json:"action"`
 	Args   []Arg  `json:"args,omitempty"`
 }
+
+// Windows contains platform specific configuration for Windows based containers.
+type Windows struct {
+	// Resources contains constraints for containers.
+	Resources *WindowsResources `json:"resources,omitempty"`
+	// Networking contains the network connection settings for containers.
+	Networking *Networking `json:"networking,omitempty"`
+	// FirstStart is an optimization on first boot of a container.
+	FirstStart bool `json:"first_start,omitempty"`
+	// LayerFolder is the path to the current layer folder.
+	LayerFolder string `json:"layer_folder,omitempty"`
+	// Layer contains the filepaths of the parent layers.
+	LayerPaths []string `json:"layer_paths,omitempty"`
+	// Settings relating to Hyper-V containers, omitted otherwise.
+	HvRuntime *HvRuntime `json:"hv_runtime,omitempty"`
```

Comment:
This is probably best delegated to `containerd` or some higher layer.  See previous discussion about the very-similar `HypervisorPath` (since dropped from #405) [here][1] and [here][2].

[1]: https://github.com/opencontainers/runtime-spec/pull/405#discussion_r61262329
[2]: https://github.com/opencontainers/runtime-spec/pull/405#discussion_r61473810